src/test/lua/exasolvs/test_query_renderer.lua

```diff
@@ -570,6 +588,14 @@ end
 
 function test_query_renderer.test_scalar_function_in_select_list_with_two_arguments()
     local parameters = {
+        {
+            func_name = "ADD",
+            first_arg_type = "literal_exactnumeric",
+            first_arg_value = 1,
+            second_arg_type = "literal_exactnumeric",
+            second_arg_value = 1,
+            expected = "SELECT 1 + 1"
```

Comment:
```suggestion
            first_arg_value = 1,
            second_arg_type = "literal_exactnumeric",
            second_arg_value = 2,
            expected = "SELECT 1 + 2"
```
Use different values to ensure arguments are not mixed up